Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [KitchenOwl/App](https://hosted.weblate.org/projects/kitchenowl/kitchenowl/).


It also includes following components:

* [KitchenOwl/Default Items](https://hosted.weblate.org/projects/kitchenowl/default-items/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/kitchenowl/kitchenowl/horizontal-auto.svg)

mortezaPRK commented:
Hey @TomBursch 
Any idea when this is going to be merged?
I added Persian translation so that my parents could use the app 🙏

weblate committed:
Translated using Weblate (Portuguese)

Currently translated at 99.3% (661 of 665 strings)

Translated using Weblate (Slovenian)

Currently translated at 83.7% (557 of 665 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 23.4% (70 of 298 strings)

Translated using Weblate (Slovenian)

Currently translated at 53.6% (357 of 665 strings)

Translated using Weblate (Greek)

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Greek)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Slovenian)

Currently translated at 35.7% (238 of 665 strings)

Translated using Weblate (Slovenian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Hebrew)

Currently translated at 90.5% (602 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 97.9% (292 of 298 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 4.6% (14 of 298 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Portuguese)

Currently translated at 98.7% (657 of 665 strings)

Translated using Weblate (Persian)

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Persian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Persian)

Currently translated at 0.3% (1 of 298 strings)

Added translation using Weblate (Persian)

Added translation using Weblate (Persian)

Translated using Weblate (Italian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (French)

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (French)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (French)

Currently translated at 98.9% (295 of 298 strings)

Translated using Weblate (French)

Currently translated at 98.9% (295 of 298 strings)

Translated using Weblate (Dutch)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 1.0% (3 of 298 strings)

Translated using Weblate (Ukrainian)

Currently translated at 97.3% (290 of 298 strings)

Translated using Weblate (Ukrainian)

Currently translated at 97.3% (290 of 298 strings)

Added translation using Weblate (Chinese (Traditional Han script))

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (665 of 665 strings)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translated using Weblate (Ukrainian)

Currently translated at 99.6% (663 of 665 strings)

Translated using Weblate (Ukrainian)

Currently translated at 99.6% (663 of 665 strings)

Translated using Weblate (Ukrainian)

Currently translated at 99.6% (663 of 665 strings)

Translated using Weblate (Ukrainian)

Currently translated at 99.6% (663 of 665 strings)

Translated using Weblate (Ukrainian)

Currently translated at 96.6% (288 of 298 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 93.2% (278 of 298 strings)

Translated using Weblate (Polish)

Currently translated at 97.9% (292 of 298 strings)

Translated using Weblate (Polish)

Currently translated at 97.9% (292 of 298 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 83.9% (558 of 665 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 83.9% (558 of 665 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 83.9% (558 of 665 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 83.9% (558 of 665 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 88.5% (264 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Croatian)

Currently translated at 100.0% (298 of 298 strings)

Translated using Weblate (Slovak)

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Slovak)

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Hebrew)

Currently translated at 70.9% (472 of 665 strings)

Translated using Weblate (Ukrainian)

Currently translated at 80.3% (534 of 665 strings)

Translated using Weblate (Ukrainian)

Currently translated at 96.3% (287 of 298 strings)

Translated using Weblate (Hungarian)

Currently translated at 100.0% (665 of 665 strings)

Translated using Weblate (Polish)

Currently translated at 96.6% (288 of 298 strings)

Translated using Weblate (Hungarian)

Currently translated at 100.0% (298 of 298 strings)

Co-authored-by: Alex Bergsland <alexbergsland@proton.me>
Co-authored-by: Amestris <dragonsteeel@gmail.com>
Co-authored-by: André Almeida <andre.myspace@gmail.com>
Co-authored-by: Anita Aaa <anita.33672@gmail.com>
Co-authored-by: David Elazar <arcdud@gmail.com>
Co-authored-by: Eduardo San Juan <ext.eduardo.sanjuan@tree-it.de>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Jerry Chou <ms0242808@gmail.com>
Co-authored-by: Krchen <kallen2004kallen@protonmail.com>
Co-authored-by: Lubos <lubos@me.com>
Co-authored-by: Ma Grz <magrz.ekst@gmail.com>
Co-authored-by: Marcelo Lago <marcelosantoslago@gmail.com>
Co-authored-by: Morteza Pourkazemi <mortz.prk@gmail.com>
Co-authored-by: Nours <mcorniere.abo@gmail.com>
Co-authored-by: Oleksii <seller.suza@gmail.com>
Co-authored-by: Pieter V <pieter@vandeven.fr>
Co-authored-by: Snir hamelc <tukan105@gmail.com>
Co-authored-by: Ste <stefano.sattanino@gmail.com>
Co-authored-by: Tine <tine.sivec7@gmail.com>
Co-authored-by: mgabor <mgabor@users.noreply.hosted.weblate.org>
Co-authored-by: rrondo <rrondoo@gmail.com>
Co-authored-by: ssantos <ssantos@web.de>
Co-authored-by: zipping <allin1+5639vrnf@d7fe0f4bc4da.8shield.net>
Co-authored-by: Олег <oleh02@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/el/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/fa/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/fr/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/he/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/hu/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/pt/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/sk/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/sl/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/default-items/uk/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/el/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/es/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/fa/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/fr/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/he/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/hr/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/hu/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/it/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/nl/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/pl/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/pt/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/sl/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/sv/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/uk/
Translate-URL: https://hosted.weblate.org/projects/kitchenowl/kitchenowl/zh_Hant/
Translation: KitchenOwl/App
Translation: KitchenOwl/Default Items

TomBursch commented:
@mortezaPRK thanks for adding the translation. Sorry it took so long. I had to prepare https://github.com/TomBursch/kitchenowl/pull/870 as well for this. I'll create a new release with it this week.

If you encounter any right-to-left text issues in the app, please take a screenshot and open an issue for it 🙂
